time-to-live.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Periodically Delete Data Using Time to Live
+summary: Use Time to Live to automatically expire and delete old data.
+---
+
+# Periodically Delete Data Using Time to Live
+
+Time to live (TTL) is a feature that allows you to manage TiDB data lifetime at the row level. For a table with the TTL attribute, TiDB automatically checks data lifetime and deletes expired data at the row level. This feature can effectively save storage space and enhance performance in some scenarios.
+
+The following are some common scenarios for TTL:
+
+* Regularly delete verification codes and short URLs.
+* Regularly delete unnecessary historical orders.
+* Automatically delete intermediate results of calculations.
+
+TTL is designed to help users clean up unnecessary data periodically and in a timely manner without affecting the online read and write workloads. TTL concurrently dispatches different jobs to different TiDB nodes to delete data in parallel in the unit of table. TTL does not guarantee that all expired data is deleted immediately, which means that even if some data is expired, the client might still read that data some time after the expiration time until that data is deleted by the background TTL job.
+
+> **Warning:**
+>
+> This is an experimental feature. It is not recommended that you use it in a production environment.
+> TTL is not available for [TiDB Cloud Serverless Tier](https://docs.pingcap.com/tidbcloud/select-cluster-tier#serverless-tier-beta).
```

Comment:
Add a note to https://docs.pingcap.com/tidbcloud/serverless-tier-limitations/?

Reply:
Thanks for the reminder. I'll add a note when tidb cloud documentation switches to v6.5 (presumably next week).